server/bin/pbench-audit-server.py

```diff
@@ -185,14 +173,14 @@ def verify_tar_dirs(ihier, tarball_dirs, tblist, controller):
 
 
 # 'ihier' is IncomingHierarchy class object
-def verify_incoming(ihier, verifylist, hierarchy=False):
+def verify_incoming(ihier, verifylist):
 
     for controller in verifylist:
         ihier.add_controller(controller)
         direct_entries = glob.iglob(
             os.path.join(ihier.config.INCOMING, controller, "*")
         )
-        tarball_dirs, unpacking_tarball_dirs = (list() for i in range(2))
+        tarball_dirs, unpacking_tarball_dirs = list(), list()
```

Comment:
I didn't really mean to make a big deal of your "curious idiom" but as long as we're stuck on that long and winding road... I think at this point it'd probably be easier to read on a quick scan if you just divided these into two separate assignments!

Reply:
Sure, no problem